### PR TITLE
Duplicate Forbidden exception that WILL get logged to Sentry

### PR DESCRIPTION
**Pull request by omgitsbillryan**
Connects: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3993

### CODE REVIEW

**Comment by patrickvinograd** [comment]
Nice. I was planning to do something just like this for `RecordNotFound` - add `RecordNotFound` to the sentry skip list, and introduce a `UnexpectedRecordNotFound` subclass that components can use if 404s are actually exceptional. In other words I like the precedent.

File: lib/common/exceptions/internal/evss_forbidden.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+module Common
```
Comment (kreek) [inline lib/common/exceptions/internal/evss_forbidden.rb]: should probably live in the lib/evss dir (with that namespacing) as it's not common, or if it is common that we want a 'visible' forbidden exception maybe rename it.

Comment (patrickvinograd) [inline lib/common/exceptions/internal/evss_forbidden.rb]: Maybe "UnexpectedForbiddenException" to line up with proposed "UnexpectedRecordNotFound"?

Comment (omgitsbillryan) [inline lib/common/exceptions/internal/evss_forbidden.rb]: I like the name change for this PR and will implement.  This gets me thinking that the architecture should be the other way around: base exceptions should always be logged and there should be some sub-folder (or maybe a Mixin?) like "non_loggable" and the `SKIP_SENTRY_EXCEPTION_TYPES` logic would exclude all of those.

**Comment by saneshark** [comment]
I think this makes sense, but you could also introduce an attribute that is defaulted differently for each error class, that could be overridden with options, specifying whether or not it should be logged.

**Pull request merged**